src/AccountServices.js

```diff
@@ -0,0 +1,147 @@
+var RemoteRoom = require("matrix-appservice-bridge").RemoteRoom;
+var RemoteUser = require("matrix-appservice-bridge").RemoteUser;
+var Twitter    = require('twitter');
+var log        = require('npmlog');
+var OAuth      = require('oauth');
+var TwitterHandler = require('./TwitterHandler.js').TwitterHandler;
+
+
+
+/**
+ * Construct a AccountServices.
+ * This class is a handler for conversation between users and the bridge bot to
+ * link accounts together
+ *
+ * @class
+ * @extends {external:TwitterHandler}
+ *
+ * @param  {matrix-appservice-bridge.Bridge}   bridge
+ * @param  app_auth OAuth authentication information
+ * @param  app_auth.consumer_key Twitter consumer key
+ * @param  app_auth.consumer_secret Twitter consumer secret
+ * @param  {TwitterDB}       storage
+ * @param  {MatrixTwitter}   twitter
+ */
+var AccountServices = function (bridge, app_auth, storage, twitter) {
+  TwitterHandler.call(this,bridge,null,"service");
+  this._app_auth = app_auth;
+  this._storage = storage;
+  this._twitter = twitter;
+  this._oauth = new OAuth.OAuth(
+    'https://api.twitter.com/oauth/request_token',
+    'https://api.twitter.com/oauth/access_token',
+    this._app_auth.consumer_key,
+    this._app_auth.consumer_secret,
+    '1.0A',
+    "oob",
+    'HMAC-SHA1'
+  );
+}
+
+AccountServices.prototype.processInvite = function (event, request, context){
+  var intent = this._bridge.getIntent();
+  var event = request.getData();
+  intent.join(event.room_id).then( () => {
+    var rroom = new RemoteRoom("service_"+event.sender);
+    rroom.set("twitter_type", "service");
+    this._bridge.getRoomStore().linkRooms(context.rooms.matrix,rroom);
+    intent.sendMessage(event.room_id,{"body":"This bot can help you link/unlink your twitter account with the Matrix Twitter Bridge. If this was not your intention, please kick the bot.","msgtype":"m.text"});
+    //Add the room to the list.
+  });
+};
+
+AccountServices.prototype.processMessage = function (event, request, context){
+  var intent = this._bridge.getIntent();
+  if(event.content.body == "link account"){
+    log.info("Handler.AccountServices",event.sender + " is requesting a twitter account link.");
```

Comment:
Given we're using ES6 arrow functions, we should make liberal use of [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) as well:

```javascript
log.info(`Handler.AccountServices ${event.sender} is requesting a twitter account link.`);
```